src/test/java/seedu/address/model/CatalogTest.java

```diff
@@ -109,45 +112,41 @@ public void getAvailableBooks_noBooksInCatalog_returnEmptyList() {
         assertTrue(loanedBooks.size() == 0);
     }
 
-    //    @Test
-    //    public void getLoanedBooks_typicalCatalogWithLoanedBooks_returnsListOfLoanedBooks() {
-    //        Catalog newData = getTypicalCatalog();
-    //        SerialNumber currentSerialNumber = new SerialNumber("B00007");
-    //        BorrowerId currentBorrowerId = new BorrowerId(VALID_BORROWER_ID);
-    //
-    //        Loan currentLoan =
-    //                new Loan(currentSerialNumber,
-    //                        currentBorrowerId,
-    //                        DateUtil.getTodayDate(),
-    //                        DateUtil.getTodayPlusDays(30));
-    //
-    //        Book loanedBook = new BookBuilder().withSerialNumber("B00007").withLoan(currentLoan).build();
-    //        newData.addBook(loanedBook);
-    //        catalog.resetData(newData);
-    //        ObservableList<Book> loanedBooks = catalog.getLoanedBooks();
-    //        assertTrue(loanedBooks.size() == 1);
-    //        assertTrue(loanedBooks.get(0).equals(loanedBook));
-    //    }
-
-    //    @Test
-    //    public void getAvailableBooks_typicalCatalogWithLoanedBooks_returnsListOfAvailableBooks() {
-    //        Catalog newData = getTypicalCatalog();
-    //        SerialNumber currentSerialNumber = new SerialNumber("B00007");
-    //        BorrowerId currentBorrowerId = new BorrowerId(VALID_BORROWER_ID);
-    //
-    //        Loan currentLoan =
-    //                new Loan(currentSerialNumber,
-    //                        currentBorrowerId,
-    //                        DateUtil.getTodayDate(),
-    //                        DateUtil.getTodayPlusDays(30));
-    //
-    //        Book loanedBook = new BookBuilder().withSerialNumber("B00007").withLoan(currentLoan).build();
-    //        newData.addBook(loanedBook);
-    //        catalog.resetData(newData);
-    //        ObservableList<Book> availableBooks = catalog.getAvailableBooks();
-    //        assertTrue(availableBooks.size() == 4);
-    //        assertTrue(getTypicalCatalog().getBookList().equals(availableBooks));
-    //    }
+    @Test
+    public void getLoanedBooks_typicalCatalogWithLoanedBooks_returnsListOfLoanedBooks() {
+        Catalog newData = getTypicalCatalog();
+        LoanId loanId = new LoanId(VALID_LOAN_ID);
+        SerialNumber currentSerialNumber = new SerialNumber(VALID_SERIAL_NUMBER_BOOK_4);
+        BorrowerId currentBorrowerId = new BorrowerId(VALID_BORROWER_ID);
+
+        Loan currentLoan = new Loan(loanId, currentSerialNumber, currentBorrowerId,
+                DateUtil.getTodayDate(), DateUtil.getTodayPlusDays(30));
```

Comment:
Would you consider declaring `30` as a final constant? Maybe it could be `private static final int LOAN_PERIOD = 30`